### PR TITLE
Plugin Generator Updates

### PR DESCRIPTION
**Pull request by Fryguy**
Plugin Generator Updates

### CODE REVIEW

**Comment by Fryguy** [comment]
@agrare Please review

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -3,27 +3,26 @@
...
+  secrets = Rails.application.secrets
   secrets.each_key do |provider|
     next if %i(secret_key_base secret_token).include?(provider) # Defaults
+
     cred_hash = secrets.public_send(provider)
     cred_hash.each do |key, value|
       config.filter_sensitive_data("#{provider.upcase}_#{key.upcase}") { CGI.escape(value) }
       config.filter_sensitive_data("#{provider.upcase}_#{key.upcase}") { value }
     end
   end
```
Comment (Fryguy) [inline spec/spec_helper.rb:27]: I feel like this secrets thing should be in the code provider generator.  This way, every plugin can follow the same pattern if this is the one we like.  Also, I'm not sure why config/secrets.yml is not in the .gitignore like I see in the Azure plugin (and I'm not sure why the Azure plugin doesn't have this code).  @agrare What do you think?

Comment (Fryguy) [reply spec/spec_helper.rb:27]: If not in the generator, then maybe a spec/support/shared file?

Comment (agrare) [reply spec/spec_helper.rb:27]: I don't think `config/secrets.yml` is pluggable so I don't know why azure has `config/secrets.yml` in its `.gitignore`

**Review by agrare — APPROVED** [review]

**Pull request merged**